homeassistant/components/onvif/camera.py

```diff
@@ -93,59 +70,37 @@
     }
 )
 
-SERVICE_PTZ_SCHEMA = vol.Schema(
-    {
-        ATTR_ENTITY_ID: cv.entity_ids,
-        vol.Optional(ATTR_PAN): vol.In([DIR_LEFT, DIR_RIGHT]),
-        vol.Optional(ATTR_TILT): vol.In([DIR_UP, DIR_DOWN]),
-        vol.Optional(ATTR_ZOOM): vol.In([ZOOM_OUT, ZOOM_IN]),
-        ATTR_MOVE_MODE: vol.In([CONTINUOUS_MOVE, RELATIVE_MOVE, ABSOLUTE_MOVE]),
-        vol.Optional(ATTR_CONTINUOUS_DURATION, default=0.5): cv.small_float,
-        vol.Optional(ATTR_DISTANCE, default=0.1): cv.small_float,
-        vol.Optional(ATTR_SPEED, default=0.5): cv.small_float,
-    }
-)
-
 
 async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
```

Comment:
You can remove this, that message is printed automatically if no platform setup is found.

Reply:
If this isn't here, it isn't importing the yaml into a config flow.